extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -107,7 +109,10 @@ export function doubleclickIsA4AEnabled(win, element) {
     experimentInfoMap[DOUBLECLICK_A4A_EXPERIMENT_NAME] = {
       isTrafficEligible: () => true,
       branches: [DOUBLECLICK_EXPERIMENT_FEATURE.HOLDBACK_INTERNAL_CONTROL,
-        DOUBLECLICK_EXPERIMENT_FEATURE.HOLDBACK_INTERNAL],
+                 DOUBLECLICK_EXPERIMENT_FEATURE.HOLDBACK_INTERNAL,
```

Comment:
This should be a separate experiment name with separate call to randomlySelectUnsetExperiments.  You can use its results explicitly to determine FF eligibility (no need to include with other experiment selection) as those are isolated to AMP cache pages.